services/net/content/ContentManager.cs

```diff
@@ -294,6 +314,88 @@ private string FixHTMLIssues(string? text)
         return text.Replace("<![CDATA[", "").Replace("]]>", "").Replace("]]&gt;", "");
     }
 
+    /// <summary>
+    /// Get settings list.
+    /// If settings list exists in memory cache, get it from memory cache.
+    /// </summary>
+    /// <returns></returns>
+    private IEnumerable<API.Areas.Services.Models.Setting.SettingModel>? GetSettings()
+    {
+        lock (_settingsLock)
+        {       
+            if (!_memoryCache.TryGetValue(SettingsListCacheKey, out SettingsList))
+            {
+                SettingsList = this.Api.GetSettings().Result;
+                _memoryCache.Set(SettingsListCacheKey, SettingsList, TimeSpan.FromMinutes(30));
```

Comment:
How do you plan to deal with changes made in the application?  With this it would ignore all changes for at least 30 minutes.  One of the areas that might change more often is topics when importing content from TNO.

Reply:
you can restart the service, which will clear the memory cache, if you need the changes in effect right away.

Reply:
Not exactly an ideal solution, and one that will result in future oddities when content is not imported due to failures of out of sync data.